mysql-test/extra/binlog_tests/binlog_delete_with_compress_column.test

```diff
@@ -0,0 +1,20 @@
+#
+# PS-7806: Column compression breaks async replication on PS
+#
```

Comment:
The tests pass even without fix, so please add a replication test (rpl suite) as this bug was initially found in a replication scenario.
```
------------------------------------------------------------------------------
[ 14%] binlog.binlog_row_delete_with_compress_column 'mix'  [ skipped ]  Doesn't support --binlog-format = 'mixed'
[ 28%] binlog.binlog_stm_delete_with_compress_column 'mix'  [ pass ]   4712
[ 42%] binlog.binlog_row_delete_with_compress_column 'row'  [ pass ]   5768
[ 57%] binlog.binlog_stm_delete_with_compress_column 'row'  [ skipped ]  Doesn't support --binlog-format = 'row'
[ 71%] binlog.binlog_row_delete_with_compress_column 'stmt'  [ skipped ]  Doesn't support --binlog-format = 'statement'
[ 85%] binlog.binlog_stm_delete_with_compress_column 'stmt'  [ pass ]   4456
[100%] shutdown_report                           [ pass ]
```